src/utils.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union
+from io import BytesIO
+from PIL import Image
+
+import numpy as np
+
+from viam.logging import getLogger
+from viam.media.video import CameraMimeType
+from viam.media.video import ViamImage
+
+LOGGER = getLogger(__name__)
+
+SUPPORTED_IMAGE_TYPE = [
```

Comment:
mild preference to put an `S` on the end of the name: several types appear to be supported.